setup.cfg

```diff
@@ -57,5 +57,6 @@ docs =
     sphinx
     numpydoc
     pydata-sphinx-theme
```

Comment:
If pydata-sphinx-theme is broken, should the reference be removed?

Reply:
Good call. Done